packages/vulcan-forms/lib/components/Form.jsx

```diff
@@ -119,7 +128,18 @@ class Form extends Component {
 
   */
   getDocument = () => {
-    const document = merge({}, this.state.initialDocument, this.defaultValues, this.state.currentValues);
+    const deletedValues = {};
+    this.state.deletedValues.forEach(path => {
+      set(deletedValues, path, null);
+    });
```

Comment:
is there a reason we're `set`ting things to `null` instead of using lodash's `unset`?